### DISCUSSION TITLE
Rating and popularity service

### DISCUSSION THREAD

**Discussion by ricardogpinho** [OnionUI/Onion#1271] [Ideas]
It would be nice to sort Roms by popularity or by rating. This could require an onionOS account or somethimg, but if you could gather everyone's rating and playtime (trough the activity tracker) we could sort the Roms accordingly. I think that a very small server could handle this.

**Comment by oherau** [comment]
The screen scrapper could retreive those data as well as other info (summary, trivia, dev team, etc.)  
The retroachievement API is free and can easily give those info.
Moreover retroachievement credentials might be already set for trophies and can be also be used for that purpose.
